core/controllers/editor.py

```diff
@@ -294,6 +309,8 @@ def _get_exploration_data(self, exploration_id, version=None):
             'tags': exploration.tags,
             'title': exploration.title,
             'version': exploration.version,
+            'is_version_of_draft_valid': is_version_of_draft_valid,
+            'draft_changes': draft_changes
```

Comment:
No, you will also need it when the version is valid in order to initialise the front-end change list.

- So that the user can see they have unsaved changes. As they make more changes these will be added to the change list, and then when they save the whole thing will be sent to the backend save handler.

 - In the draft_changes key.